src/icons/fileIcons.ts

```diff
@@ -890,5 +890,6 @@ export const fileIcons: FileIcons = {
         { name: 'commitlint', fileNames: ['.commitlintrc', '.commitlintrc.js', 'commitlint.config.js', '.commitlintrc.json', '.commitlint.yaml', '.commitlint.yml'] },
         { name: 'buck', fileNames: ['.buckconfig'] },
         { name: 'dhall', fileExtensions: ['dhall', 'dhallb'] },
+        { name: 'typeorm', fileNames: ['ormconfig.json', 'ormconfig.js', 'ormconfig.ts'] },
```

Comment:
If you want to provide an icon for the light theme you have to add the `light` property here and set it to true.

Reply:
Done